src/utils/service_builder.rs

```diff
@@ -108,24 +110,31 @@ impl ServiceBuilder {
             return Err(Error::new(ErrorKind::InvalidData, "need one provider"));
         }
 
-        let backend_handlers = build_backend_handlers(providers)?;
+        // The authenticators supported by the Parsec service.
+        let mut authenticators: Vec<(AuthType, Authenticator)> = Vec::new();
+        authenticators.push((AuthType::Direct, Arc::from(DirectAuthenticator {})));
```

Comment:
I guess it does not matter now because we currently only have one authenticator, but is everything in place here to keep the list ordered if another authenticator is added?

Reply:
Yes -- the order here is important. The order in which the elements are added here is the order in which they will be returned to any client requesting them. Do you want me to add a comment here to make this clearer? Or perhaps we could extract this out into its own function?

Reply:
I think a comment would be good, to note for the future that the order here is important!

Reply:
Done!